galaxy/templates/deployment-celery-beat.yaml

```diff
@@ -0,0 +1,207 @@
+            failureThreshold: {{ .Values.webHandlers.readinessProbe.failureThreshold }}
+            timeoutSeconds: {{ .Values.webHandlers.readinessProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.celeryBeat.livenessProbe.enabled }}
+          livenessProbe:
+            exec:
+              command: [
+                "bash",
+                "-c",
+                "/galaxy/server/.venv/bin/celery -A galaxy.celery inspect ping -d celery@$HOSTNAME"
+              ]
+            initialDelaySeconds: {{ .Values.webHandlers.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.webHandlers.livenessProbe.periodSeconds }}
+            failureThreshold: {{ .Values.webHandlers.livenessProbe.failureThreshold }}
+            timeoutSeconds: {{ .Values.webHandlers.livenessProbe.failureThreshold }}
+          {{- end }}
+          volumeMounts:
+            {{- range $key, $entry := .Values.extraFileMappings -}}
+            {{- if $entry.applyToCelery }}
+            - name: {{ include "galaxy.getExtraFilesUniqueName" $key }}
+              mountPath: {{ $key }}
+              subPath: {{ include "galaxy.getFilenameFromPath" $key }}
+            {{- end }}
+            {{- end }}
+            {{- range $key,$entry := .Values.configs }}
+            {{ if $entry -}}
+            - name: galaxy-conf-files
+              mountPath: /galaxy/server/config/{{ $key }}
+              subPath: {{ $key }}
+            {{- end -}}
+            {{- end }}
+            {{- range $key,$entry := .Values.jobs.rules }}
+            - name: galaxy-job-rules
+              mountPath: /galaxy/server/lib/galaxy/jobs/rules/{{ $key }}
+              subPath: {{ $key }}
+            {{- end }}
+            - name: galaxy-data
+              mountPath: {{ .Values.persistence.mountPath }}
+            - name: galaxy-data
+              mountPath: /galaxy/server/config/mutable/
+              subPath: config
+            {{- if .Values.setupJob.downloadToolConfs.enabled }}
+            - name: galaxy-data
+              mountPath: {{ .Values.setupJob.downloadToolConfs.volume.mountPath }}
+              subPath: {{ .Values.setupJob.downloadToolConfs.volume.subPath }}
+            {{- end }}
+            {{- if .Values.refdata.enabled }}
+            {{- range $key, $entry := .Values.refdata.galaxyPersistentVolumeClaims }}
+            - name: refdata-gxy-{{ $key }}
+              mountPath: {{ $entry.mountPath }}
+            {{- end -}}
+            {{- end }}
+            {{- if .Values.extraVolumeMounts }}
+            {{- tpl (.Values.extraVolumeMounts | toYaml | nindent 12) . }}
+            {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+        # - name: nginx
+        #   image: nginx
+        #   lifecycle:
+        #     preStop:
+        #       exec:
+        #         command: ["/usr/sbin/nginx","-s","quit"]
```

Comment:
Why have this here? If left, we should add a comment when it should be uncommented.

Reply:
Totally missed this. Seems to have been copied over from the web handler. Removed all.